src/svg/bar.js

```diff
@@ -1,112 +1,13 @@
-import d3 from 'd3';
+import { rebindAll } from '../util/rebind';
+import { path as svgPath } from 'd3-path';
+import { bar as barPath } from 'd3fc-path';
 
-// Renders a bar series as an SVG path based on the given array of datapoints. Each
-// bar has a fixed width, whilst the x, y and height are obtained from each data
-// point via the supplied accessor functions.
+// Renders an OHLC as an SVG path based on the given array of datapoints. Each
+// OHLC has a fixed width, whilst the x, open, high, low and close positions are
+// obtained from each point via the supplied accessor functions.
 export default function() {
```

Comment:
I think this is one of the rare occasions that expressing it as a one-liner is actually simpler to understand? E.g.
```js
return barPath(svgPath)
```